lib/react_client/react_interop.dart

```diff
@@ -35,21 +32,21 @@ typedef JsPropValidator = dynamic Function(
 @JS()
 abstract class React {
   external static String get version;
-  external static ReactElement cloneElement(ReactElement element, [JsMap props, dynamic children]);
+  external static ReactElement cloneElement(ReactElement element, [JsMap? props, Object? children]);
   external static ReactContext createContext([
     dynamic defaultValue,
-    int Function(dynamic currentValue, dynamic nextValue) calculateChangedBits,
+    int Function(dynamic currentValue, dynamic nextValue)? calculateChangedBits,
   ]);
   @Deprecated('For internal use only.')
   external static ReactClass createClass(ReactClassConfig reactClassConfig);
   @Deprecated('Use createElement instead. To be removed in 7.0.0.')
   external static ReactJsComponentFactory createFactory(type);
-  external static ReactElement createElement(dynamic type, props, [dynamic children]);
+  external static ReactElement createElement(dynamic type, props, [Object? children]);
   external static JsRef createRef();
   external static ReactClass forwardRef(Function(JsMap props, dynamic ref) wrapperFunction);
   external static ReactClass memo(
     dynamic wrapperFunction, [
-    bool Function(JsMap prevProps, JsMap nextProps) areEqual,
+    bool Function(JsMap prevProps, JsMap nextProps)? areEqual,
```

Comment:
Does this mean that someone could pass null as the second optional argument? Would that serve a purpose?

Reply:
In Dart null safety, any optional argument without a default needs to be nullable, and as a result, yeah it's possible/valid for someone to call this with a `null` second argument.